### PR TITLE
User roles

### PR DESCRIPTION
**Pull request by seiferteric**

### CODE REVIEW

File: src/translib/authorize.go [hunk 1]
```diff
@@ -40,30 +40,41 @@ func init() {
...
+// 	if ((user == "") || (user == "admin")) {
+// 		return true
+// 	}
 
-	return false
-}
+// 	return false
+// }
 func isAuthorizedForSet(req SetRequest) bool {
 	// TODO: Allow empty user for now
```
Comment (jeffyin-dell) [inline src/translib/authorize.go]: Can delete this comment if we're getting rid of the "empty user" allowance

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
Just minor comments surrounding deleting code vs commenting out

File: src/translib/authorize.go (hunk 1 above)
Comment (seiferteric) [reply src/translib/authorize.go]: Deleted

File: src/translib/authorize.go [hunk 2]
```diff
@@ -40,30 +40,41 @@ func init() {
 }
 
 //TODO:Subscribe to UserDB changes and repopulate the cache on any notification
-func isUserAuthorizedForSet(user string) bool {
-	//TODO:Need code to get role from username and then authorize based on that
-	//This is a temporary code, this needs to be fixed
+// func isUserAuthorizedForSet(user string) bool {
```
[the remark this answers is not in the document]
Comment (seiferteric) [reply src/translib/authorize.go]: Removed

**Review by martin-belanger — APPROVED** [review]

**Review by jeffyin-dell — APPROVED** [review]

**Pull request merged**